frontends/web/src/components/status/status.tsx

```diff
@@ -80,46 +82,41 @@ export default class Status extends Component<Props, State> {
         });
     }
 
-    public render(
-        {
-            children,
-            type = 'warning',
-            dismissable,
-            className,
-        }: RenderableProps<Props>,
-        {
-            show,
-        }: State) {
-        const childrenList = children as ComponentChild[];
-        if (!show || (childrenList && childrenList.length === 1 && !childrenList[0])) {
+    public render({
+        children,
+        className,
+        dismissable,
+        hidden,
+        type = 'warning',
+    }: RenderableProps<Props>,
+    {
+        show,
+    }: State) {
+        if (hidden || !show) {
```

Comment:
@x1ddos I had to keep both `show` from state (updated after config is read) and `hidden` from prop, but got at least rid of all the children checking.